src/plugins/vis_type_table/README.md

```diff
@@ -1 +1,35 @@
-Contains the data table visualization, that allows presenting data in a simple table format.
\ No newline at end of file
+# Data Table
+
+This is an OpenSearch Dashboards plugin that is used to visualize data and aggregations in tabular format.
+
+## Create Data Table
+To create a data table in OpenSearch Dashboards, first select `Visualize` from the navigation menu. Then click `Create Visualization` and choose `Data Table` as the visualization type.
+
+## Select Metrics
+
+### Metrics Aggregation
+At the `Metrics`, select the metric aggregation type from the menu and config it accordinly.
+
+### Buckets Aggregation
+At the `Buckets`, config the columns to be displayed in the table visualization.
+- `Split Rows` is used when you want to divide one row into more categories. It splits one row into multiple and add columns based on the categories you choose to split. For example, if you split the data based on gender then you want to know more on each gender's clothing preference. You could click `Split Rows` and input `clothing.category` in terms. Each genger's data is now splitted to multiple rows based on a new added column `clothing.category`.
```

Comment:
some rewording suggestions:
```suggestion
- `Split Rows` is used when you want to divide one row into more based on some category. It splits one row into multiple and add columns based on the categories you choose to split. For example, if you split the data based on gender then you want to know more on each gender's clothing preference. You could click `Split Rows` and input `clothing.category` in terms. Each gender's data is now split across to multiple rows based on a new added column `clothing.category`.
```

But I'm still not sure this is quite right. In general, choosing split rows doesn't generally add columns. It just applies your existing columns to whatever field you split by.

Reply:
Will use the suggested rewording.

For the second question, from my observation, it always add columns when we split rows. I think the meaning of `split rows` in terms aggregation is to split on row into multiple based on the new split option. 

For example, we could have split rows with term aggregation. We could count column is split to two based on day_of_week.
<img width="732" alt="Screen Shot 2022-12-19 at 16 15 57" src="https://user-images.githubusercontent.com/79961084/208552658-c08619cd-9030-4a82-a872-c9ec8c2a872d.png">

Continue, split row again with term aggregation of field category.keyword. Now each row is split into multiple based on diff category. A new `category.keyword` col is added.
<img width="731" alt="Screen Shot 2022-12-19 at 16 15 44" src="https://user-images.githubusercontent.com/79961084/208552749-ee7b83d7-112e-4ccb-8297-148ae8ecfbd4.png">

It is not just term aggregation. Rows can be split via other aggregations as well, like Date Histogram, Histogram and etc, as long as this bucket aggregation has a field. The only bucket aggregation which doesn't have a field is `filter`. With setting a filter, it will add a filter col.

* Filter bucket agg

<img width="1045" alt="Screen Shot 2022-12-19 at 16 28 14" src="https://user-images.githubusercontent.com/79961084/208553740-8b897124-d05e-4e1a-8316-cb4862b9f6e1.png">

* Range bucket agg
<img width="735" alt="Screen Shot 2022-12-19 at 16 25 37" src="https://user-images.githubusercontent.com/79961084/208553780-f2f4f3d2-057c-42e0-9218-c896bb70f9bb.png">

* Histogram
<img width="737" alt="Screen Shot 2022-12-19 at 16 24 37" src="https://user-images.githubusercontent.com/79961084/208553839-0ffab04a-5a16-431a-97cc-725ef51b431e.png">

So I think it should be okay to remind user that `split row` will split one row and add a column to the table.

Reply:
You're right, that when splitting rows, we always need to add a column for identifying the value of the agg that we're splitting on.